.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Bug report
+name: "\U0001F41B Bug Report"
```

Comment:
If we change the word to title case, please make all the templates same. "Feature Request", "Get Support".